yarn.lock

```diff
@@ -13,9 +13,9 @@
     string-loader "0.0.1"
     vue-clickaway "^2.1.0"
 
-"@aeternity/aepp-sdk@github:aeternity/aepp-sdk-js#feature/sweet-api":
-  version "0.14.0-0.1.0-alpha.1"
-  resolved "https://codeload.github.com/aeternity/aepp-sdk-js/tar.gz/8dbad4eb72004115f9bbf1d822b8460a36ad8df1"
+"@aeternity/aepp-sdk@github:aeternity/aepp-sdk-js#develop":
+  version "0.15.0-0.1.0"
+  resolved "https://codeload.github.com/aeternity/aepp-sdk-js/tar.gz/7cda8877e9f69a6584105749febc10ccb7dfbb89"
```

Comment:
New versions of this packages are related to changes in this commit?

Reply:
@davidyuk no. I'm using yarn and npm at the same time due to some issues, with yarn not building dependencies, for the `ae-components`